docs/quick-start.md

```diff
@@ -53,3 +53,96 @@ python manage.py update_vector_indexes
 ```
 
 To skip the prompt, use the `--noinput` flag.
+
+## Using event-stream (WagtailVectorIndexSSEConsumer)
```

Comment:
Not quite sure this belongs in the "quick-start" section, might be better off in it's own markdown file